### DISCUSSION TITLE
Suggestion: Changing the background of markers

### DISCUSSION THREAD

**Discussion by Firefly65536** [keiwando/toonsquid#374] [Ideas]
Allow us to change the background of markers, I think this should be relatively simple to make that will help

![60AA4B65-63FB-4F09-8C26-039CAAEDF9F9](https://github.com/keiwando/toonsquid/assets/121312948/574c3757-81f6-4ac7-8dbe-960513c56e60)
I have eye assets that are purely white and I can't see which one is which

**Comment by keiwando** [comment]
Makes sense. Thanks for the feedback!
